ndcube/ndcube.py

```diff
@@ -1112,12 +1116,11 @@ def my_propagate(uncertainty, data, mask, **kwargs):
         new_unit = new_unit or self.unit
         # Make sure the input bin dimensions are integers.
         bin_shape = np.rint(bin_shape).astype(int)
-        offsets = (bin_shape - 1) / 2
         if all(bin_shape == 1):
             return self
         # Ensure bin_size has right number of entries and each entry is an
         # integer fraction of the array shape in each dimension.
-        data_shape = self.dimensions.value.astype(int)
+        data_shape = self.shape
```

Comment:
Does this need to be an array?  Or is leaving it as a tuple OK?  I'm looking at [L1127](https://github.com/sunpy/ndcube/pull/684/files#diff-929794bb532be1486907b360285433b432f8505ab7e7ff63eca3eab657f7d00dR1127)

Reply:
Testing in the terminal, tuple inputs returns a array